pages/tutorials/supabase-search.mdx

```diff
@@ -0,0 +1,402 @@
+search indices, documents, and configurations within the intuitive BuildShip environment.
+
+The Supabase Nodes provide a no-code interface for engaging with Supabase, ensuring a hassle-free integration of
+advanced search features into your projects. From indexing new documents and tweaking search settings to conducting
+intricate search queries, Supabase Nodes are designed to streamline these processes in a user-friendly manner.
+
+## Prerequisites ✅
+
+Before you can start using the Supabase Nodes in BuildShip, ensure that you have the following prerequisites in place:
+
+### Supabase Project
+
+You'll need access to a Supabase project. Follow these steps to prepare your account and initiate your first project:
+
+1. Navigate to [Supabase](https://supabase.com/) to either sign up for a new account or log in if you are an existing
+   user.
+2. Create a new project and configure your database as needed.
+
+It's important to note, generating API keys for secure interaction with your Supabase instance is an important step that
+follows the establishment of your project.
+
+### API Keys
+
+Supabase utilizes API keys to authenticate, granting complete operational access. For enhanced security, it is
+recommended to [generate new API keys](https://supabase.com/docs/guides/api) with restricted scopes tailored to specific
+use cases.
+
+### Host
+
+The URL of your Supabase instance serves as the host. This critical piece of information is required for connecting
+BuildShip's Supabase nodes to your Supabase server. For more details on obtaining your Supabase instance's URL, refer to
+the [Supabase documentation](https://supabase.com/docs/guides/api).
+
+For comprehensive guidance on setting up the Supabase project, please refer to the
+[Supabase Tutorial](https://docs.buildship.com/tutorials/supabase-crud).
+
+For the Full Text Search, Semantic Search, and Hybrid Search nodes, suppose you have a Supabase table called `documents`
+with the columns `id`, `title`, `content`, and `created_at`. You can follow along with this guide to try out the nodes.
+
+You can create the documents table with the following SQL code:
+
+```sql
+CREATE TABLE documents (
+    id SERIAL PRIMARY KEY,
+    title TEXT NOT NULL,
+    content TEXT NOT NULL,
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+);
+```
+
+## Full Text Search
+
+The "Full Text Search" node allows you to perform full-text searches on a Supabase table, enabling you to find documents
+that match a specific search query.
+
+<Image src={supabase6} alt='Supabase' width={1200} />
+
+### Prerequisites
+
+1. Supabase Project: Ensure you have a Supabase project set up.
+2. Table with Data: Ensure your table has data added as described in the
+   ["Creating Data"](https://docs.buildship.com/tutorials/supabase-crud#creating-data) section.
+
+### Step-by-Step Guide
+
+1. Add Full Text Search Column
```

Comment:
Perhaps we can use the Steps component for this to make it look more step-like? (Similarly for all the guides on this page) Example: https://docs.buildship.com/trigger-nodes/rest-api#add-trigger-node

Also, not sure if its meaningful to specify the Prerequisites again as we already have a section for that. If we have any specific information to mention we can use the Callout components instead.